src/VCR/Util/StreamProcessor.php

```diff
@@ -280,22 +280,23 @@ public function stream_tell()
      *
      * @link http://www.php.net/manual/en/streamwrapper.url-stat.php
      *
-     * @param  string $path The file path or URL to stat.
+     * @param  string  $path  The file path or URL to stat.
+     * @param  integer $flags Holds additional flags set by the streams API.
      *
-     * @return integer      Should return as many elements as stat() does.
+     * @return integer        Should return as many elements as stat() does.
      */
-    public function url_stat($path)
+    public function url_stat($path, $flags)
     {
         $this->restore();
-        try {
+        if ($flags & STREAM_URL_STAT_QUIET) {
+            set_error_handler(function() {
+                // Use native error handler
+                return false;
+            });
             $result = @stat($path);
-        } catch (\ErrorException $e) {
```

Comment:
Looks nice thank you. I wonder though, if we don't catch the exception PHPUnit might throw ErrorExceptions when running in `processIsolation`. Did you test this by any chance?